### ISSUE TITLE
ArrayPattern is not supported

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#499]
# Language
JavaScript

# Sample input source code
https://github.com/piotrwitek/react-redux-typescript-guide/blob/master/playground/src/hoc/with-error-boundary.usage.tsx
```
  const [shouldRenderBrokenComponent, setShouldRenderBrokenComponent] =
useState(false);
```

# Expected output
N/A

# Actual output (or error message)
In help mode
```
 - ArrayPattern - {"line":14,"column":8} - {"line":14,"column":69}
[shouldRenderBrokenComponent, setShouldRenderBrokenComponent]
```

**Comment by k2snowman69** [comment]
Will release with 2.6.3 or greater

**Comment by k2snowman69** [comment]
Released with 3.0.0
